frontend/src/pages/cellguide/[cellTypeId].tsx

```diff
@@ -1,3 +1,11 @@
+/**
```

Comment:
The intention is to use `[cellTypeId].tsx` to serve both `CELL_GUIDE_CELL_TYPE` and `CELL_GUIDE_TISSUE_SPECIFIC_CELL_TYPE` routes, so we have single source of truth and not duplicate code where possible